app/static/playlist.style.css

```diff
@@ -118,6 +119,46 @@ p:last-child {
   color: #fff;
 }
 
+/* COLLECT - BOTTOM RIGHT */
```

Comment:
Seeing there aren't any other comments in the CSS, you can probably remove this.

Reply:
Yeah agreed.